hw3/02-url.js

```diff
@@ -18,21 +18,34 @@ const server = http.createServer((req, res) => {
     routes.forEach(
       (elem) => (result += `<li><a href="${elem}">${elem}</a></li>`)
     );
-
     return result;
   };
 
   if (req.url === '/') {
     let routeResults = getRoutes();
-
     res.writeHead(200, { 'Content-Type': 'text/html' });
     res.write(`<h1>Exercise 02</h1>`);
-
     res.write(`<ul> ${routeResults} </ul>`);
   }
 
   // Add your code here
-
+  //Check if query parameters exist in the URL.
+  else if (url.search.length > 0) {
+    //Create a string variable to be able to append strings based on the queries.
+    let htmlString = '';
+    //Search for each value and associated key in the url query parameters.
+    url.searchParams.forEach((value, key) => {
+      //Append to the string a table row and column in html.
+      htmlString += `<tr> <td> ${key} </td> <td> ${value} </td> </tr>`;
+    });
+    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.write(`<table border = "1"> ${htmlString} </table>`);
```

Comment:
I think this idea is nice. Wrap them in the expression and put them in the template literal like this makes the code very easy to follow.